indigo_app/views/places.py

```diff
@@ -50,20 +53,40 @@ def get_context_data(self, **kwargs):
         return context
 
 
+# TODO: WorkFilterForm
 class PlaceDetailView(PlaceViewBase, AbstractAuthedIndigoView, TemplateView):
     template_name = 'place/detail.html'
     js_view = 'LibraryView'
     tab = 'works'
 
+    def get(self, request, *args, **kwargs):
+        params = QueryDict(mutable=True)
+        params.update(request.GET)
```

Comment:
This is where we need to set defaults, in particular:

* exclude stubs
* draft + published
* all subtypes
* sorting: most recently updated